packages/components/src/InputDate/InputDate.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { InputDate } from ".";
+
+afterEach(cleanup);
+
+it("renders a blank form by default", () => {
+  const { getByDisplayValue } = render(<InputDate onChange={jest.fn()} />);
+  expect(getByDisplayValue("")).toBeTruthy();
+});
+
+it("fires onChange with the new value when you click a date", () => {
+  const date = "11/11/2011";
+  const newDate = "11/15/2011";
+  const changeHandler = jest.fn();
+  const { getByDisplayValue, getByText } = render(
+    <InputDate value={new Date(date)} onChange={changeHandler} />,
+  );
+
+  const form = getByDisplayValue(date);
+  fireEvent.focus(form);
+
+  const selectDate = getByText("15");
+  fireEvent.click(selectDate);
+  expect(changeHandler).toHaveBeenCalledWith(new Date(newDate));
```

Comment:
very minor thing.

Line `25`, `44` both matches the value with `new Date()` but line `76` puts it in a const. There's nothing wrong with it, but y'know consistency 😉

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/1e3029a931ed8275f06edb2b1bbdea83d2000589